<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>izpispodatkov</class>
 <widget class="QDialog" name="izpispodatkov">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtri</string>
   </property>
   <widget class="QCheckBox" name="checkBoxSpol">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Spol</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxKategorija">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kategorija</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Prika≈æi</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>691</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
